docs/DeveloperGuide.adoc

```diff
@@ -121,13 +121,13 @@ image::ModelClassDiagram.png[]
 
```

Comment:
Might want to remove the arrows in composition and aggregation. 
i.e.
![2](https://user-images.githubusercontent.com/54141404/67541982-7cb72380-f71d-11e9-9393-bb50bc0081f5.PNG)
to:
![3](https://user-images.githubusercontent.com/54141404/67542103-fb13c580-f71d-11e9-978c-eb7a62d60194.PNG)

Reply:
If QuestionList implements ReadOnlyQuestionList then ReadOnlyQuestionList should be an abstract class or an interface.
![1](https://user-images.githubusercontent.com/54141404/67541762-bb98a980-f71c-11e9-80ec-f3115c58f866.PNG)